app/controllers/api/generic_object_definitions_controller.rb

```diff
@@ -99,7 +99,7 @@ def remove_methods_resource(type, id, data)
 
     def self.allowed_association_types
       GenericObjectDefinition::ALLOWED_ASSOCIATION_TYPES.collect do |association_type|
-        [Dictionary.gettext(association_type, :type => :model, :notfound => :titleize, :plural => false), association_type]
+        [Dictionary.gettext(association_type, :type => :model, :notfound => :titleize, :plural => false, :translate => false), association_type]
```

Comment:
what about the other call to gettext in that file in the self.allowed_typed method ? maybe that too should get a :translate => false ?

Reply:
I don't see a gettext call in `self.allowed_types` method.

Sure, `GenericObjectDefinition::TYPE_NAMES` does contain `N_()` in its definition, but `N_()` is quite harmless actually. It's just an identity function: no real translation is being made, it's just a marker for the gettext string collection process (to put the string inside `N_()` to a catalog). In fact, it's just this use case (delayed translation), where we're supposed to use `N_()` during declaration and then `_()` at the moment of translation.

Reply:
I saw it in the gaprindashvili branch, didn't know if that was an issue.

Reply:
Strictly speaking, it is an issue (for the reasons we already discussed) and can be resolved by cherry-picking the following 2 PRs into gaprindashvili: https://github.com/ManageIQ/manageiq/pull/16563 and https://github.com/ManageIQ/manageiq-api/pull/232

Reply:
got it, so keep this PR g/yes then ?

Reply:
Well, if we want to resolve this problem for gaprindashvili as well, then yes, we need to cherry-pick all the following PRs into gaprindashvili: https://github.com/ManageIQ/manageiq/pull/16563 https://github.com/ManageIQ/manageiq-api/pull/232 https://github.com/ManageIQ/manageiq-ui-classic/pull/2962 and the current one as well.

Reply:
@abellotti So should I label https://github.com/ManageIQ/manageiq/pull/16563 and https://github.com/ManageIQ/manageiq-api/pull/232 as g/yes ?

Reply:
I think so, so gapringashvili on it's consistent.   @martinpovolny thoughts ?